sycamore/writers/file_writer.py

```diff
@@ -166,7 +166,7 @@ def write(
             fs = _unwrap_s3_serialization_workaround(filesystem)
 
             for row in block.iter_rows(public_row_format=True):
-                doc = Document(row)
+                doc = Document.deserialize(row.get("doc"))
```

Comment:
This seems like a common enough thing. Should we just have a Document.from_row(row) to avoid repeating the .get("doc") thing everywhere?